charts/values.yaml

```diff
@@ -117,3 +117,16 @@ startupProbe:
   failureThreshold: 30
   successThreshold: 1
 
+logging:
```

Comment:
Let's change "logging" to "logCollection" as we don't want to imply that this turns on or off actually logging.

Reply:
P.S. Sorry to have missed this in the spec review. We should have got the names right there.